client/src/main/java/org/spine3/client/ColumnFilters.java

```diff
@@ -0,0 +1,237 @@
+    }
+
+    /**
+     * Creates new "greater than" {@link ColumnFilter}.
+     *
+     * <p>For the supported types description see <a href="#types">Comparision types section</a>.
+     *
+     * @param columnName the name of the Entity Column to query by, expressed in a single field
+     *                   name with no type info
+     * @param value      the requested value of the Entity Column
+     * @return new instance of ColumnFilter
+     */
+    public static ColumnFilter gt(String columnName, Object value) {
+        checkNotNull(columnName);
+        checkNotNull(value);
+        checkSupportedOrderingComparisonType(value.getClass());
+        return createFilter(columnName, value, GREATER_THAN);
+    }
+
+    /**
+     * Creates new "less than" {@link ColumnFilter}.
+     *
+     * <p>For the supported types description see <a href="#types">Comparision types section</a>.
+     *
+     * @param columnName the name of the Entity Column to query by, expressed in a single field
+     *                   name with no type info
+     * @param value      the requested value of the Entity Column
+     * @return new instance of ColumnFilter
+     */
+    public static ColumnFilter lt(String columnName, Object value) {
+        checkNotNull(columnName);
+        checkNotNull(value);
+        checkSupportedOrderingComparisonType(value.getClass());
+        return createFilter(columnName, value, LESS_THAN);
+    }
+
+    /**
+     * Creates new "greater or equal" {@link ColumnFilter}.
+     *
+     * <p>For the supported types description see <a href="#types">Comparision types section</a>.
+     *
+     * @param columnName the name of the Entity Column to query by, expressed in a single field
+     *                   name with no type info
+     * @param value      the requested value of the Entity Column
+     * @return new instance of ColumnFilter
+     */
+    public static ColumnFilter ge(String columnName, Object value) {
+        checkNotNull(columnName);
+        checkNotNull(value);
+        checkSupportedOrderingComparisonType(value.getClass());
+        return createFilter(columnName, value, GREATER_OR_EQUAL);
+    }
+
+    /**
+     * Creates new "less or equal" {@link ColumnFilter}.
+     *
+     * <p>For the supported types description see <a href="#types">Comparision types section</a>.
+     *
+     * @param columnName the name of the Entity Column to query by, expressed in a single field
+     *                   name with no type info
+     * @param value      the requested value of the Entity Column
+     * @return new instance of ColumnFilter
+     */
+    public static ColumnFilter le(String columnName, Object value) {
+        checkNotNull(columnName);
+        checkNotNull(value);
+        checkSupportedOrderingComparisonType(value.getClass());
+        return createFilter(columnName, value, LESS_OR_EQUAL);
+    }
+
+    /**
+     * Creates new conjunction aggregating filter.
```

Comment:
Please use search feature of your IDE to locate all `aggregat?* filter` entries within this PR and remove those forever :)